src/lib/constants/routes.js

```diff
@@ -0,0 +1,18 @@
+export const Routes = {
+  DROPSITE_DETAIL: (dropsiteId) => `/dropsite/${dropsiteId}`,
+  DROPSITE_ADMIN: (dropsiteId) => `/dropsite/${dropsiteId}/admin`,
+  DROPSITE_NEW_ADMIN: (dropsiteId) => `/dropsite/new/admin/${dropsiteId}`,
+  FAQ: '/faq',
+  HOME: '/',
+  LOGIN: '/login',
+  LOGOUT: '/logout',
+  PENDING_DOMAINS: '/pending-domains',
+  PROFILE: '/profile',
+  NEW_FACILITY: '/new-facility',
+  REQUEST_SERVICES: '/request-services',
+  REQUEST_SUPPLIES: '/request',
+  SIGNUP_DROPSITE: (dropsiteId) => `/signup/${dropsiteId}`,
+  SIGNUP_FINISH_DROPSITE: (dropsiteId) => `/signupFinish/${dropsiteId}`,
```

Comment:
It'd be better if `Routes` was always an object of strings, so it's more predictable.

I also don't see a problem having the route params in this file.

```suggestion
  SIGNUP_FINISH_DROPSITE: `/signupFinish/:dropsiteId?`,
```

I'd also prefer a generic function that takes any string and fills the parameters, something like:
```ts
routeWithParams(Routes.SIGNUP_FINISH_DROPSITE, { dropsiteId });
```

Thoughts?

Reply:
Love it!